_delphi_utils_python/data_proc/geomap/geo_data_proc.py

```diff
@@ -531,6 +533,56 @@ def derive_zip_to_state_code():
     )
 
 
+def derive_fips_hhs_crosswalk():
+    """
+    Builds a crosswalk between FIPS county codes and HHS regions.
+    """
+    if not isfile(join(OUTPUT_DIR, STATE_HHS_OUT_FILENAME)):
+        create_state_hhs_crosswalk()
+    if not isfile(join(OUTPUT_DIR, FIPS_POPULATION_OUT_FILENAME)):
+        create_fips_population_table()
+
+    fips_pop = pd.read_csv(
+        join(OUTPUT_DIR, FIPS_POPULATION_OUT_FILENAME), dtype={"fips": str, "pop": int}
+    )
+    state_hhs = pd.read_csv(
+        join(OUTPUT_DIR, STATE_HHS_OUT_FILENAME),
+        dtype={"state_code": str, "state_id": str, "state_name": str},
```

Comment:
Looks like the state -> HHS file should have column names `state_code` and `hhs_region_number`. Same problem on line 576.

Reply:
> Have you considered leftpadding the HHS region numbers with zero so that they're all the same length? It can make string processing and sorting more convenient.

I went with the existing implementation (state -> hhs) that had them without the zero padding, which also follows the existing HRR convention of not left padding. I think it's to distinguish geos that are defined with zero pad (e.g. a zip code "06001" is defined as 5 digits and "6001" wouldn't be right, same for a county FIPS like  "01009") from those without (e.g. [HHS Regions](https://www.hhs.gov/about/agencies/iea/regional-offices/index.html) are just "Region 1."  I don't feel super strongly either way, though if we left padded HHS we should also probably do for HRR and update any downstream documentation accordingly.